pycroft/lib/finance.py

```diff
@@ -829,12 +829,11 @@ def build_transactions_query(account, search=None, sort_by='valid_on', sort_orde
 def match_activities():
     """Get a dict of all unmatched transactions and a user they should be matched with
 
-    :param BankAccount bank_account: The BankAccount to get the unmatched transactions from
-
     :returns: Dictionary with transaction and user
     :rtype: Dict
```

Comment:
This rtype now outdated and should be replaced by a proper type annotation.
However, using dicts to transport structured data is a bad idea anyway (not your fault).
The dicts should be replaced by `@dataclass` types.

This allows someone else to more easily because the IDE can help you with completion, and because there is a central locatiion where one can look up what data is _actually_ being passed (namely the type definition).